python/cudf/cudf/core/index.py

```diff
@@ -1906,9 +1906,9 @@ def find_label_range(self, first, last):
         col = self._values
         begin, end = None, None
         if first is not None:
-            begin = col.find_first_value(first, closest=True)
+            begin = int(col.find_first_value(first, closest=True))
         if last is not None:
-            end = col.find_last_value(last, closest=True)
+            end = int(col.find_last_value(last, closest=True))
```

Comment:
What types are these before casting to `int`?

Reply:
`float`(index location is correct but the type isn't for example instead of `6` the returned value is `6.0`) incase the input is a floating number

Reply:
Instead of typecasting here, should https://github.com/rapidsai/cudf/blob/branch-0.15/python/cudf/cudf/utils/cudautils.py#L151 be created as an `int32` array instead of the type of the input array?

Reply:
Done 👍